### PR TITLE
React Capstone Project

### PR DESCRIPTION
**Pull request by bill7pearl**
# In this PR I have built a react app that fetches a list of free to play videogames, I have:
### Home page:
- When the page loads, the webapp shows the list of games that could be filtered out by typing the game name.
- Along with the game name, a numeric values are displayed from from the API (release date).
- When the user clicks (or taps) on a game, the application navigates to the details page.
### Details page:
- In the details page, the webapp retrieves data from the API to show detailed data of the game.
- When the user clicks on the "Back" button (<), the user navigates to the home page.
### Testing:
- Created unit tests for pure functions in my Redux code.
- Created integration tests using the React Testing Library.
- Mocked the access to the API.
- Mocked the connection to the Redux Store.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,160 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
```
Comment (t-yanick) [inline README.md:150]: - [x] Great job working on this capstone project.  As per project requirements, you are supposed to give appropriate credit to Nelson Sakwa the author of the original design, as required by the Creative Commons License.

File: src/index.css [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+.details-title {
+  color: red;
+}
+
+.details-content {
+  color: black;
+  font-size: 0.9rem;
+}
+
+@media screen and (min-width: 780px) {
```
Comment (t-yanick) [inline src/index.css]: - [x] As per the given design, in mobile view   there are two columns with alternating colors.  At the moment your app displays the games in just one column.  Please kindly make it two columns.

**Review by t-yanick — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED 

Hi @bill7pearl , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Great video presentation.  I am so proud of what you achieved 
:heavy_check_mark: Project well structured and Use of Redux for state management 
:heavy_check_mark: Good number of unit and integration tests added.
:heavy_check_mark: Awesome use of `map` to retrieve needed data
:heavy_check_mark: App has two pages as per project requirements 
:heavy_check_mark: Use of `react-router-dom`
:heavy_check_mark: App deployed and live link works well

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by t-yanick — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED 

Hi @bill7pearl , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Great video presentation.  I am so proud of what you achieved 
:heavy_check_mark: Project well structured and Use of Redux for state management 
:heavy_check_mark: Good number of unit and integration tests added.
:heavy_check_mark: Awesome use of `map` to retrieve needed data
:heavy_check_mark: App has two pages as per project requirements 
:heavy_check_mark: Use of `react-router-dom`
:heavy_check_mark: App deployed and live link works well

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.css (hunk 2 above)
Comment (bill7pearl) [reply src/index.css]: ### done

fixedfeedback

File: README.md (hunk 1 above)
Comment (bill7pearl) [reply README.md:150]: done

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @bill7pearl ,
# STATUS: APPROVED! 🥇 💯 
done

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**